app/code/Magento/Vault/composer.json

```diff
@@ -1,15 +1,18 @@
 {
     "name": "magento/module-vault",
     "description": "",
+    "config": {
+        "sort-packages": true
+    },
     "require": {
         "php": "7.0.2|7.0.4|~7.0.6|~7.1.0",
         "magento/framework": "100.3.*",
-        "magento/module-sales": "100.3.*",
-        "magento/module-store": "100.3.*",
         "magento/module-checkout": "100.3.*",
-        "magento/module-payment": "100.3.*",
         "magento/module-customer": "100.3.*",
+        "magento/module-payment": "100.3.*",
         "magento/module-quote": "100.3.*"
+        "magento/module-sales": "100.3.*",
```

Comment:
Seems to be a miss `,` the line above.

Reply:
Thank you for catching these, @dmanners!